lib/utils.js

```diff
@@ -146,3 +150,56 @@ utils.registerSourceMap = () => {
 utils.registerTypeScript = () => {
   require('ts-node').register();
 };
+
+/**
+ * Check if given template is installed and return details about it
+ * 
+ * @private
+ * @param {String} name name of the template
+ * @param {String} PACKAGE_JSON_FILENAME standard name of the package.json file
+ * @returns {Object|undefined} object representing package.json file with additional field -> pkgPath
```

Comment:
```suggestion
 * @returns {Object|undefined} object representing content of the package.json file with additional field -> pkgPath
```

Please add info that object represents content of package.json to clarify :)